wiki/Tournaments/ANZT/2019_Summer/en.md

```diff
@@ -0,0 +1,112 @@
+[spreadsheet]: https://docs.google.com/spreadsheets/d/1gnV5_7e-57MXtpoY57lgyPKyF3_IvjDXsGoLVgrnqk8
+
+# Aus/NZ osu!standard Tournament 6 - Summer
+
+![ANZT 6S Logo](img/logo.png)
+
+The **Aus/NZ osu!standard Tournament 6 - Summer** (***ANZT 6S***) was a single elimination one-on-one osu!standard tournament with a Swiss format group stage hosted by ![][flag_NZ] [Diony](https://osu.ppy.sh/users/4294475) and ![][flag_AU] [EmeraldStar82](https://osu.ppy.sh/users/3772135). It was the 7th installment of Aus/NZ tourneys.
```

Comment:
Is ANZT 6S really the 7th installment?

Reply:
Confusingly, yes. There was ANZT 1 through 4, then ANZT 5S & 5W, and now 6S